core/standalone/src/main/scala/org/apache/openwhisk/standalone/LogbackConfigurator.scala

```diff
@@ -39,7 +39,10 @@ object LogbackConfigurator {
 
   def initLogging(conf: Conf): Unit = {
     val ctx = LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext]
-    ctx.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME).setLevel(toLevel(conf.verbose()))
+    //Only tweak the log level if verbose option is used
+    if (conf.verbose() != 0) {
```

Comment:
does this make a difference? what is the default level otherwise on the call at line 44?

Reply:
Default log level is `INFO`. So far with current impl it was not possible to disable logging as it was always getting reset to INFO programmatically